prep-day.sh

```diff
@@ -45,9 +45,9 @@ if test -e "src/day$1.rs"; then
 else
   echo "Creating boilerplate module for day $1 at src/day$1.rs..."
   echo "Remember to update main.rs:"
```

Comment:
If you're going to make them copy-pastable lines, can you update this to say "Remember to add the following lines to main.rs"?

Reply:
Ok!